### DISCUSSION TITLE
✅ Récuperer sur l'iphone un fichier sur la machine distante.

### DISCUSSION THREAD

**Discussion by alaingrach** [isontheline/pro.webssh.net#136] [Q&A] [answered]
Bonjour,

Je n'ai pas trouver comment transférer un fichier de la machine distante sur l'iphone à partir de WebSSH. Est-ce possible ? Et si oui, comment ?

**Comment by isontheline** [comment]
Bonjour Alain 👋

Malheureusement non mais cette fonctionnalité est planifiée ici : https://github.com/isontheline/pro.webssh.net/issues/114

Pour le moment seule l'édition de fichiers texte est possible.

Votre besoin est de télécharger vers l'iPhone et de réutiliser le fichier dans une autre application iOS ?

**Reply by alaingrach** [reply]
Oui tout à fait, je voudrai, à distance, pouvoir parcourir tout mon ordinateur (ça c'est OK) et pouvoir télécharger vers mon iphone des fichiers pour les utiliser sur l'iphone.

**Comment by isontheline** [comment]
Entendu, merci pour votre retour !

Le téléchargement de fichiers devrait être l'une des priorités de la version 15.0

**Comment by isontheline** [comment]
Bonjour Alain,

Je viens d'implémenter l'upload (iOS vers serveur) : https://testflight.apple.com/join/QSrBK59z

Je vais maintenant implémenter le download : https://github.com/isontheline/pro.webssh.net/issues/87

Bonnes fêtes 🥳

**Comment by isontheline** [comment, marked as answer]
Upload and Download are now available inside [WebSSH 14.10] - Truck(https://webssh.net/documentation/changelog/14.10/)
